### DISCUSSION TITLE
How to set-up auto update for private repository

### DISCUSSION THREAD

**Discussion by cchenot** [cawa-93/vite-electron-builder#876] [Q&A] [answered]
Hi there !
I've built an electron react app with this template which is great,
I am now trying to set-up auto-update.
I have manually triggered the "release workflow" for my app, on the staging branch. And published 2 different releases (on 2 different days). On the windows PC where I've downloaded and installed the .exe, I don't see the updates of the second release.
Not sure if I am missing something or if I should add anything in the electron-builder.config.js (I haven't added anything yet regarding auto-update).
Thank you !

EDIT: my github repo is private, maybe this is the source of the issue? Thanks !

**Comment by cawa-93** [comment, marked as answer]
Yeap. Out-of-box this template configured only for public github repos. [Follow the doc](https://www.electron.build/auto-update#private-github-update-repo) to use private repo.

**Reply by cchenot** [reply]
Thanks a lot !
Also, any tips for updating my repo with your latest changes (I saw you updated the workflow with breaking changes) which pulled your template in August 2022?
Should I just modify the files manually?

**Reply by cawa-93** [reply]
Since the template mostly focuses on the development of the environment and infrastructure such as compilation, sys, testing and almost does not contain custom business logic, then You probably may extract new template and copy-paste your renderer/preload code. 
Or you may use git patches any way 🤷🏻‍♂️

**Reply by cchenot** [reply]
Thank you !

**Comment by nappalm** [comment]
@cchenot did you manage to do it? I'm a little confused with the process 🥲
